### PR TITLE
Feat: Argent one button connector UI

### PR DESCRIPTION
**Pull request by Cussone**
### Issue / feature description

Adding UI elements for 1 button connector feature

### Checklist

- [x] Rebased to the last commit of the target branch (or merged)
- [x] Code self-reviewed
- [x] Code self-tested
- [x] Tests updated (if needed)
- [x] All tests are passing locally

### CODE REVIEW

File: src/modal/ConnectorButton.svelte [hunk 1]
```diff
@@ -15,6 +16,8 @@
         ? wallet.icon.dark
         : wallet.icon.light
   const isSvg = icon?.startsWith("<svg")
+
+  console.log(wallet)
```
Comment (bluecco) [inline src/modal/ConnectorButton.svelte:20]: remove

File: src/modal/Modal.svelte [hunk 2]
```diff
@@ -1,142 +1,177 @@
...
+
   export let dappName: string = window?.document.title ?? ""
   export let modalWallets: ModalWallet[]
-  export let callback: (
-    value: StarknetkitConnector | null,
-  ) => Promise<void> = async () => {}
-  export let theme: "light" | "dark" | null = null
-
-  let loadingItem: string | false = false
+  console.log(modalWallets)
```
Comment (bluecco) [inline src/modal/Modal.svelte:17]: remove

File: src/modal/Modal.svelte [hunk 3]
```diff
@@ -1,142 +1,177 @@
 <script lang="ts">
-  import { onMount } from "svelte"
-  import type { StarknetWindowObject } from "@starknet-io/types-js"
-  import ConnectorButton from "./ConnectorButton.svelte"
-  import type { StarknetkitConnector } from "../connectors/connector"
-  import { InjectedConnector } from "../connectors/injected"
+  // import { onMount } from "svelte"
```
Comment (bluecco) [inline src/modal/Modal.svelte:2]: all these comments are placeholders or can be removed?

File: src/modal/Modal.svelte [hunk 4]
```diff
@@ -1,142 +1,177 @@
...
+  >
+    <Header title={dappName} showBackButton={showBackButton} />
 
-    if (isInAppBrowser && window?.starknet_argentX) {
-      try {
-        callback(new InjectedConnector({ options: { id: "argentX" } }))
-      } catch {}
-      return
-    }
+    <!--			<WalletList {walletList} />-->
```
Comment (bluecco) [inline src/modal/Modal.svelte:86]: same q for these imports

File: src/modal/Modal.svelte [hunk 5]
```diff
@@ -1,142 +1,177 @@
...
-            xmlns="http://www.w3.org/2000/svg"
-          >
-            <path
-              d="M9.77275 3.02275C9.99242 2.80308 9.99242 2.44692 9.77275 2.22725C9.55308 2.00758 9.19692 2.00758 8.97725 2.22725L6 5.20451L3.02275 2.22725C2.80308 2.00758 2.44692 2.00758 2.22725 2.22725C2.00758 2.44692 2.00758 2.80308 2.22725 3.02275L5.20451 6L2.22725 8.97725C2.00758 9.19692 2.00758 9.55308 2.22725 9.77275C2.44692 9.99242 2.80308 9.99242 3.02275 9.77275L6 6.79549L8.97725 9.77275C9.19692 9.99242 9.55308 9.99242 9.77275 9.77275C9.99242 9.55308 9.99242 9.19692 9.77275 8.97725L6.79549 6L9.77275 3.02275Z"
-              fill="currentColor"
-            />
-          </svg>
-        </span>
-      </header>
+<!--{#if !isInAppBrowser && modalWallets.length > 1}-->
```
Comment (bluecco) [inline src/modal/Modal.svelte:102]: same q for these comments

**Review by bluecco — APPROVED** [review]
just a couple of comments, lgtm

File: src/modal/Modal.svelte (hunk 4 above)
Comment (Cussone) [reply src/modal/Modal.svelte:86]: wip

File: src/modal/Modal.svelte (hunk 3 above)
Comment (Cussone) [reply src/modal/Modal.svelte:2]: wip

File: src/modal/Modal.svelte (hunk 2 above)
Comment (Cussone) [reply src/modal/Modal.svelte:17]: wip

File: src/modal/ConnectorButton.svelte (hunk 1 above)
Comment (Cussone) [reply src/modal/ConnectorButton.svelte:20]: wip

File: src/modal/Modal.svelte (hunk 5 above)
Comment (Cussone) [reply src/modal/Modal.svelte:102]: wip

**Pull request merged**